notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    def vms_data_to_pandas(filename, vms_type, begin_time=None, end_time=None):
</pre><p>Please, add two empty lines before the definition of a function. </p><p>Same for the other functions in this cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>